spec/bolt/analytics_spec.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt/analytics'
+
+describe Bolt::Analytics do
+  let(:default_config) { {} }
+
+  before :each do
+    # We use a hard override to disable analytics for tests, but that obviously
+    # interferes with these tests...
+    ENV.delete('BOLT_DISABLE_ANALYTICS')
```

Comment:
Is this reverted after the test? Seems like it might need to be `:around`.

Reply:
Oh, we reapply it before each test.

Reply:
The spec_helper sets it in a `before :each`, so it should be reset for future tests.